app/controllers/api/v1/courses_controller.rb

```diff
@@ -24,6 +24,11 @@ def create
     end
   end
 
+  def destroy
+    @course = course.find(params[:id])
+    @course.destroy
+  end
```

Comment:
I will suggest you return something to indicate a successful deletion of the course.
You can add a `header :no_content` after destroying the course.

Reply:
Hi @enkog,

Thanks for your detailed review! I did all of the required changes. You can review it now, please